src/components/Banner/Banner.stories.tsx

```diff
@@ -185,7 +189,9 @@ export const DismissableWithAction: StoryObj<Args> = {
 export const DismissableBelowContent: StoryObj<Args> = {
   render: (args) => (
     <>
-      <Heading size="h1">Page Title</Heading>
+      <Heading className={styles.headingBottomSpacing} size="h1">
```

Comment:
nit: do we still want to use bracket notation for styles in stories?

Reply:
Good call, will update 👍